docs/user_guide/plugins.rst

```diff
@@ -0,0 +1,4 @@
+.. _user-plugins:
+
+User plugin guide
+=================
```

Comment:
I added it to be able to reference a user's guide when I was writing the developer's guide so the latter could be finalized even though there isn't a real user guide yet. I think the title states the intention. To be more explicit, it's about how to install plugins, where to get them, how to customize them etc.